src/routes/namespaces/[namespace]/workflows/index.svelte

```diff
@@ -43,18 +43,20 @@
     <header>
       <WorkflowFilters bind:timeFormat />
     </header>
-    <WorkflowsSummaryTable>
-      <tbody slot="rows">
-        {#await data}
-          <WorkflowsLoadingState />
-        {:then { workflows }}
-          {#each workflows as workflow}
-            <WorkflowsSummaryRow {workflow} {timeFormat} />
-          {:else}
-            <WorkflowsEmptyState />
-          {/each}
-        {/await}
-      </tbody>
-    </WorkflowsSummaryTable>
+    {#await data}
+      <WorkflowsLoadingState />
+    {:then { workflows }}
+      {#if workflows.length > 0}
```

Comment:
Tiny nit… you can just do `workflows.length` since zero is falsey and everything else is truthy.